R/path-explorer.R

```diff
@@ -66,7 +66,7 @@ getSimulationTree <- function(simulationOrFilePath, quantityType = "Quantity") {
   }
 
   # Build a vector, with no duplicated entries, of all paths corresponding to entities in `simulation` that fall under the types specified in quantityType
-  allPaths <- sapply(quantityType,function(type){quantityTypeList[[type]](simulation)}) %>% unname %>% unlist %>% unique
+  allPaths <- unique(unlist(unname(sapply(quantityType,function(type){quantityTypeList[[type]](simulation)}))))

```

Comment:
Why removing pipes? I learnt from @IndrajeetPatil that it might be more efficient since it does not duplicate the data in memory (maybe I am completely wrong)

Reply:
Lol I wasn't sure what to take away from #700: is the consensus that we should use pipes or not?  I couldn't find a single instance of pipes in the whole package, so I thought we're leaving them out for now.  Personally I prefer pipes too in cases like this.

Reply:
Pipe all the way....there is a big PR with lots of pipe coming